### DISCUSSION TITLE
Alternative USB dongle with radio signal processing in hardware

### DISCUSSION THREAD

**Discussion by KDYR** [wmbusmeters/wmbusmeters#843] [Q&A] [answered]
Currently running wmbusmeters with a SDR dongle, but that is implying elevated CPU load and temperature on my fan-less RPi. Was looking for an iM871A dongle, but they seem impossible to buy anywhere (found only one vendor with the unit on stock, but they only sell b2b, i.e. VAT registration is required). So therefore: 
Anyone who knows of an alternative USB dongle with radio signal processing in hardware and wmbusmeters support?

**Comment by nobodyguy** [comment]
Check out my iM871A board :)
[https://github.com/nobodyguy/wmbus_hardware](https://github.com/nobodyguy/wmbus_hardware)

**Reply by KDYR** [reply]
thanks @nobodyguy 
That is interesting, but as I only need one, it would be quite a project to order pcb and components. Further, with access to an old-fashioned hand-held soldering iron only, I don't think I could even mount the components. 
Are there plans for someone manufacturing a batch and then selling individual dongles?

**Reply by nobodyguy** [reply]
Sure, let's switch to emails :) jarynznosa@gmail.com

**Comment by Tpaydt** [comment]
> Currently running wmbusmeters with a SDR dongle, but that is implying elevated CPU load and temperature on my fan-less RPi. Was looking for an iM871A dongle, but they seem impossible to buy anywhere (found only one vendor with the unit on stock, but they only sell b2b, i.e. VAT registration is required). So therefore: Anyone who knows of an alternative USB dongle with radio signal processing in hardware and wmbusmeters support?

Hello,

i have one and could sell it to you, but i live in germany.

**Comment by avandorp** [comment, marked as answer]
I've bought mine here: https://shop.imst.de/wireless-modules/usb-radio-products/10/im871a-usb-wireless-m-bus-usb-adapter-868-mhz Was a bit expensive as I don't live in Germany but otherwise no issues.
